packages/tinacms/src/toolkit/fields/plugins/blocks-field-plugin/index.tsx

```diff
@@ -278,7 +278,7 @@ const InvalidBlockListItem = ({
   return (
     <Draggable
       key={index}
-      type={field.name}
+      // type={field.name}
```

Comment:
**question**: Why is this commented-out code retained?

Reply:
removed